examples/storm-starter/src/clj/org/apache/storm/starter/clj/word_count.clj

```diff
@@ -60,18 +61,22 @@
 
 (defn mk-topology []
 
-  (topology
-   {"1" (spout-spec sentence-spout)
-    "2" (spout-spec (sentence-spout-parameterized
-                     ["the cat jumped over the door"
-                      "greetings from a faraway land"])
-                     :p 2)}
-   {"3" (bolt-spec {"1" :shuffle "2" :shuffle}
-                   split-sentence
-                   :p 5)
-    "4" (bolt-spec {"3" ["word"]}
-                   word-count
-                   :p 6)}))
+  (Thrift/buildTopology
+   {"1" (Thrift/prepareSpoutDetails sentence-spout)
+    "2" (Thrift/prepareSpoutDetails (sentence-spout-parameterized
+                                    ["the cat jumped over the door"
+                                     "greetings from a faraway land"])
+                                    (Integer. 2))}
+   {"3" (Thrift/prepareBoltDetails {(Utils/getGlobalStreamId "1" nil)
+                                    (Thrift/prepareShuffleGrouping)
+                                    (Utils/getGlobalStreamId "2" nil)
+                                    (Thrift/prepareShuffleGrouping)}
+                                    split-sentence
+                                    (Integer. 5))
+    "4" (Thrift/prepareBoltDetails {(Utils/getGlobalStreamId "3" nil)
+                                    (Thrift/prepareFieldsGrouping ["word"])}
+                                   word-count
+                                   (Integer. 6))}))
```

Comment:
This is one case where we are breaking compatibility by moving to java and I am not sure we should do this.  I think we still want to have/support a clojure API, but we may need a separate maven package to do that, and thrift.clj is a big part of that API. Could you please inline the needed thrift.clj functions inside clojure.clj instead of deleting them?
```
(defalias topology thrift/mk-topology)
(defalias bolt-spec thrift/mk-bolt-spec)
(defalias spout-spec thrift/mk-spout-spec)
(defalias shell-bolt-spec thrift/mk-shell-bolt-spec)
(defalias shell-spout-spec thrift/mk-shell-spout-spec)
```